tutorials/fluid_modeling.ipynb

```diff
@@ -0,0 +1,935 @@
+{
```

Comment:
<p>The first paragraph here is a partial answer to the question 'how can we simulate such systems', or 'what happens under the hood in such simulations' - but not necessary to answer how a non-trivial fluid can be modeled in PorePy. Both types of information is relevant, and both can be fit here, but the plan for when to communicate which type is not clear to me.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Let's discuss in person.</p>